corehq/sql_db/util.py

```diff
@@ -263,6 +263,16 @@ def filter_out_stale_standbys(dbs):
     ]
 
 
+def weighted_choice(choices, weights):
+    assert len(choices) == len(weights), 'Every choice needs a weight'
+    assert 0.9 < sum(weights) < 1.1, 'Weights should be normalized'
+
+    big_weights = (int(w * 100) for w in weights)
```

Comment:
`int(w * 100)` will effectively drop choices whose normalized weight is less than 0.01. If we want to avoid that we could use `math.ceil`, but I don't think it's a concern for our usecase.

Reply:
Seems like this might not always be obvious and use cases tend to change over time. Maybe add a docstring with this comment?

Reply:
I'm also somewhat against this as these weights are provided by localsettings changes that are defined in commcare-cloud. I think currently its fine, but there could be a case where we want the main database to have < 1 % of reads and the standbys to handle all read load (unless the standbys are down/behind) so main can focus on write. (I think this point is null if you go with the py3 implementation though)